x-pack/filebeat/input/httpjson/config_request.go

```diff
@@ -102,7 +102,7 @@ type requestConfig struct {
 func (c *requestConfig) Validate() error {
 	c.Method = strings.ToUpper(c.Method)
 	switch c.Method {
-	case "POST":
+	case "POST": ///nolint:goconst // Bad linter! These constant literals are far more meaningful than any label.
 	case "GET":
```

Comment:
Maybe we should use the constants from the http package?

```suggestion
	case http.MethodPost:
	case http.MethodGet:
```